cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1691,11 +1689,12 @@ function start-kube-apiserver {
     params+=" --etcd-keyfile=${ETCD_APISERVER_CLIENT_KEY_PATH}"
   fi
   params+=" --secure-port=443"
-  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" != "true" ]]; then
+  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "false" ]]; then
     # Default is :8080
     params+=" --insecure-port=0"
-  else
-    params+=" --insecure-bind-address=0.0.0.0 --insecure-port=8080"
+  fi
```

Comment:
comparing with the default of this to-be-deprecated arg,  it is preferable to explicitly set.

Reply:
It defaults to 8080. No need to set it explicitly. I tested it and it works.